nio.conf

```diff
@@ -271,6 +278,16 @@ port=[[PKS_PORT]]
 # use localhost rather than unix domain socket
 #
 # use_localhost=False
+#
+# If the name of this section does actually point to the module path where the brew 
+# class can be found, you can specify it using the namespace configuration option
+#namespace=pubkeeper.brew.local.brew.LocalBrew
```

Comment:
This isn't directly a namespace, but rather the actual brew class.  Maybe change naming to be `brew_class` or something?

Reply:
True, `namespace` was the term used in the existing code so I kept it but you're right, class name works better here